src/main/java/duke/Main.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import duke.javaFX.MainWindow;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+    private Duke duke = new Duke();
+
+    public Main() throws IOException {
+    }
+
+    /**
+     * Starts the scene builder.
+     * @param stage The stage for the scene.
+     */
+    @Override
+    public void start(Stage stage) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = fxmlLoader.load();
+            Scene scene = new Scene(ap);
+            stage.setScene(scene);
+            fxmlLoader.<MainWindow>getController().setDuke(duke);
+            stage.show();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In general, printing stack trace is not good error handling. Think if there is anything better you can do about the exception rather than printing the stack trace.